libs/x-content/src/test/java/org/elasticsearch/common/xcontent/ObjectParserTests.java

```diff
@@ -206,7 +206,7 @@ public void setTest(int test) {
         {
             XContentParser parser = createParser(JsonXContent.jsonXContent, "{\"not_supported_field\" : \"foo\"}");
             XContentParseException ex = expectThrows(XContentParseException.class, () -> objectParser.parse(parser, s, null));
-            assertEquals(ex.getMessage(), "[1:2] [the_parser] unknown field [not_supported_field], parser not found");
+            assertEquals(ex.getMessage(), "[1:2] [the_parser] unknown field [not_supported_field]");
```

Comment:
I could preserve this bit of the message, but I don't think it was really helping anything.